java_training/src/com/lftechnology/jan7/FileInputMain.java

```diff
@@ -0,0 +1,72 @@
+package com.lftechnology.jan7;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+public class FileInputMain {
+	public static void main(String[] args) {
+		FileInput file = new FileInput();
+		file.readFile("/home/nerisa/javatest/in.txt");
+		file.displayData();
+
+	}
+
+}
+
+/**
+ * Write a program called FileInput to read an int, a double, and a String form a text file called "in.txt", and produce the following
+ * output: (Hint use Scanner to read from file) a. The integer read is 12 b. The floating point number read is 33.44 c. The String read
+ * is"Peter" d. Hi! Peter, the sum of 12 and 33.44 is 45.44
+ * 
+ * @author nerisa
+ * 
+ */
+class FileInput {
+	private static final Logger LOG = Logger.getLogger(FileInputMain.class.getName());
+	private static Scanner s = null;
```

Comment:
@anealkeshi Dai I had previously made separate classes for each operation and performed all the logic inside the main class but later I have compiled all those in the FileUtility class and FileUtilityMain class. All the other classes does have mistakes. I had already committed them so I didnot delete those classes.